firefox/guide/firefox/firefox_preferences-install_extensions/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+title: 'Extensions install must be disabled.'
```

Comment:
Please change title to `Disable Extension Installation`

Reply:
Edited the title accordingly